pom.xml

```diff
@@ -420,14 +497,25 @@
         <org.apache.cxf.version>3.3.7</org.apache.cxf.version>
         <jackson.databinding.version>2.16.1</jackson.databinding.version>
         <spring-web.version>5.1.2.RELEASE</spring-web.version>
+        <spring-web-test.version>2.5</spring-web-test.version>
+        <javax.ws.rs.version>1.1.1</javax.ws.rs.version>
+        <servlet-api.version>2.5</servlet-api.version>
+        <jstl.version>1.2</jstl.version>
+        <org.slf4j.verison>1.7.21</org.slf4j.verison>
+        <equinox.javax.servlet.version>3.0.0.v201112011016</equinox.javax.servlet.version>
+        <encoder.wso2.version>1.2.0.wso2v1</encoder.wso2.version>
+        <orbit.version.commons.httpclient>4.5.13.wso2v1</orbit.version.commons.httpclient>
+        <orbit.httpcore.version>4.3.3.wso2v1</orbit.httpcore.version>
+        <commons.io.version>2.7</commons.io.version>
+        <taglibs-standard-impl.version>1.2.5</taglibs-standard-impl.version>
+        <apache.tomcat-catalina.version>9.0.11</apache.tomcat-catalina.version>
         <encoder.wso2.version>1.2.0.wso2v1</encoder.wso2.version>
         <javax.validation.api.version>3.0.0</javax.validation.api.version>
         <hibernate-validator>7.0.1.Final</hibernate-validator>
         <commons.bean.utils.version>1.9.3</commons.bean.utils.version>
         <javax.validation.api.version>2.0.1.Final</javax.validation.api.version>
         <hibernate-validator>6.0.20.Final</hibernate-validator>
         <commons.bean.utils.version>1.9.3</commons.bean.utils.version>
```

Comment:
this also defined in L515

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/8/commits/55101612f0aa99105f83a6d7d3d648000eb13112